### ISSUE TITLE
Have hours details on graph (portfolio) on a week period.

### ISSUE THREAD

**Issue by bscholving** [LedgerHQ/ledger-live-desktop#1488]
- [x] I have checked this feature was not yet requested.

#### Portfolio and Account Page

It would be nice to have more details about the evolution of a crypto (and also global portfolio) on a graph for a week period. Maybe hours detail instead of days.

**Comment by juan-cortes** [comment]
If I'm not mistaken, this would need to be tackled on the [ledger-api-countervalue](https://github.com/LedgerHQ/ledger-api-countervalue) repo since that's where we get the counter-values for each currency, right now there's no way of getting it on an hourly manner. 

![image](https://user-images.githubusercontent.com/4631227/46262579-7139fe80-c503-11e8-9ef6-ef2cba2a7c6c.png)

What I recall from when I had to fork that repo for the stellar integration is that it relied on `histodays` which where the historical daily values for a pair of currencies.  I can't see hourly rates, although cryptocompare does provide `histohour` and `histominute`.  I don't know if the benefit of having a more granular chart justifies the work needed and the fact that the bandwidth for this would be significantly bigger too, but I digress ... work needs to be done over there before we can do anything here.

@gre should know better though

**Comment by gre** [comment]
yeah i still need to investigate all implication of such change. 
The api would have to pull both the days history and the hours history. Then in term of API exposed, either we also split in two (easy) or I also imagined an hybrid mode were we could have historical values with hour granularity for recent dates and days for more older time (It unifies one http query but this would kinda complexify the api code)

from the app logic, it's relatively easy because you just receive a `Date` object and need to project that to one of the format (try the more precise, if not available, try the less precise)

functionally I think we just need it for a week period.

**Comment by juan-cortes** [comment]
Might take a look later

**Comment by gre** [comment]
Fixed in [2.29.0](https://github.com/LedgerHQ/ledger-live-desktop/releases/tag/v2.29.0)
